src/devices/custom_devices_diy.ts

```diff
@@ -1312,77 +1318,100 @@ const definitions: Definition[] = [
-                unit: 'C',
+                unit: 'ºC',
                 cluster: 'hvacUserInterfaceCfg',
-                attribute: {ID: 0x0102, type: 40},
-                valueMin: -127,
-                valueMax: 127,
-                description: 'Comfort parameters/Temperature minimum, in 1° steps. Requires v0.1.1.7 or newer.',
+                attribute: {ID: 0x0102, type: dataType.int16},
+                valueMin: -100.0,
+                valueMax: 100.0,
+                scale: 100,
+                description: 'Comfort parameters/Temperature minimum, in 0.01°C steps.',
             }),
             numeric({
                 name: 'comfort_temperature_max',
-                unit: 'C',
+                unit: 'ºC',
                 cluster: 'hvacUserInterfaceCfg',
-                attribute: {ID: 0x0103, type: 40},
-                valueMin: -127,
-                valueMax: 127,
-                description: 'Comfort parameters/Temperature maximum, in 1° steps. Requires v0.1.1.7 or newer.',
+                attribute: {ID: 0x0103, type: dataType.int16},
+                valueMin: -100.0,
+                valueMax: 100.0,
+                scale: 100,
+                description: 'Comfort parameters/Temperature maximum, in 0.01°C steps.',
             }),
             numeric({
                 name: 'comfort_humidity_min',
                 unit: '%',
                 cluster: 'hvacUserInterfaceCfg',
-                attribute: {ID: 0x0104, type: 32},
-                valueMin: 0,
-                valueMax: 100,
-                description: 'Comfort parameters/Humidity minimum, in 1% steps. Requires v0.1.1.7 or newer.',
+                attribute: {ID: 0x0104, type: dataType.uint16},
+                valueMin: 0.0,
+                valueMax: 100.0,
+                scale: 100,
+                description: 'Comfort parameters/Humidity minimum, in 0.01% steps.',
             }),
             numeric({
                 name: 'comfort_humidity_max',
                 unit: '%',
                 cluster: 'hvacUserInterfaceCfg',
-                attribute: {ID: 0x0105, type: 32},
-                valueMin: 0,
-                valueMax: 100,
-                description: 'Comfort parameters/Humidity maximum, in 1% steps. Requires v0.1.1.7 or newer.',
+                attribute: {ID: 0x0105, type: dataType.uint16},
+                valueMin: 0.0,
+                valueMax: 100.0,
+                scale: 100,
+                description: 'Comfort parameters/Humidity maximum, in 0.01% steps.',
             }),
         ],
         ota: ota.zigbeeOTA,
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            const bindClusters = ['msTemperatureMeasurement', 'msRelativeHumidity', 'genPowerCfg'];
+            await reporting.bind(endpoint, coordinatorEndpoint, bindClusters);
+            await reporting.temperature(endpoint, {min: 10, max: 300, change: 10});
```

Comment:
This should not be needed, e.g. this line is already handled by `temperature({reporting: {min: 10, max: 300, change: 10}}),`

Reply:
Done

Reply:
The configure is still there.

Reply:
The configure is to read additional attributes optionally. They don't exist in older firmware versions.
Otherwise, there are errors in a loop and device cannot be configured.